MathLib/ODE/OdeSolverTypes.h

```diff
@@ -0,0 +1,67 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2016, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+
+#ifndef MATHLIB_ODE_ODESOLVERTYPES_H
+#define MATHLIB_ODE_ODESOLVERTYPES_H
+
+#include <Eigen/Core>
+
+namespace MathLib
+{
+template <int M, int N>
+using MappedMatrix = Eigen::Map<Eigen::Matrix<double, M, N, Eigen::ColMajor>>;
```

Comment:
Indeed, it's a general type. But it's only visible if you include the particular header (Hopefully it won't pollute all of our program.). I'd prefer to keep the shorter name, because in the ODE solver setting most users will probably not care if the matrix is row or column major; usually they'll just write to it using `operator()` or assignment to the whole matrix.
Additionally, I think we also do not indicate the storage order in the names of our other matrix types, neither for dense nor for sparse ones.